RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+#### 1.3.11 February 05 2020 ####
```

Comment:
I've set this as 1.3.xx even though a breaking change as versioning seems to follow akka.net

Reply:
👍